app/src/organisms/Desktop/Devices/Peripherals/CameraCard.tsx

```diff
@@ -97,15 +103,26 @@ export function CameraCard({
             >
               {t('on_deck')}
             </StyledText>
-            <StyledText desktopStyle="bodyDefaultRegular">
-              {t('camera')}
-            </StyledText>
+            <Flex flexDirection={DIRECTION_ROW} alignItems={ALIGN_CENTER}>
+              <Icon
+                size="1rem"
+                name="camera"
```

Comment:
```suggestion
                name="photo-camera"
```
We're on `chore_release`, so the icon is still a `photo-camera` here.